README.md

```diff
@@ -58,35 +58,39 @@ Supporting structured data doesn't mean giving up text: when Serilog writes even
 
 ### Getting started
 
-Serilog is installed from NuGet. To view log events, one or more sinks need to be installed as well, here we'll use the pretty-printing console sink, and a rolling file set:
+Serilog is installed [from NuGet](https://nuget.org/packages/serilog). To view log events, one or more sinks need to be installed as well, here we'll use the pretty-printing console sink, and a rolling file set:
 
 ```
 dotnet add package Serilog
 dotnet add package Serilog.Sinks.Console
 dotnet add package Serilog.Sinks.File
 ```
 
-The simplest way to set up Serilog is using the static `Log` class. A `LoggerConfiguration` is used to create and assign the default logger.
+The simplest way to set up Serilog is using the static `Log` class. A `LoggerConfiguration` is used to create and assign the default logger, normally in _Program.cs_:
 
 ```csharp
 using Serilog;
 
-public class Program
+Log.Logger = new LoggerConfiguration()
+    .WriteTo.Console()
+    .WriteTo.File("log.txt",
+        rollingInterval: RollingInterval.Day,
+        rollOnFileSizeLimit: true)
+    .CreateLogger();
+
+try
+{
+    // Your program here...
+    const string name = "Serilog";
+    Log.Information("Hello, {Name}!", name);
```

Comment:
Would be useful to throw an exception here to show it working?
Or perhaps a comment

Because  `Log.Information` would never throw an exception, so it's a little odd to see.

```suggestion
    Log.Information("Hello, {Name}!", name);
    throw new InvalidOperationException("Oops...");
```